### ISSUE TITLE
Can anyone reproduce the results for tiered-imagenet?

### ISSUE THREAD

**Issue by smallbox120** [Sha-Lab/FEAT#37]
I use the script and the pretrained model provided by the auther (resnet12), but only get 68.9 accuracy on 5-way 1-shot, is there anything I should notice? On the other hand, I can reproduce the results on mini-imagenet.

**Comment by Han-Jia** [comment]
Hi,

for the 1-shot ProtoNet, you can try lr=0.0001, T = 64, schedule = 20-0.5, then the model gets 83.8% accuracy.

**Comment by smallbox120** [comment]
> Hi,
> 
> for the 1-shot ProtoNet, you can try lr=0.0001, T = 64, schedule = 20-0.5, then the model gets 83.8% accuracy.

Actually 68.8% is the performance of FEAT model, and the pre-trained model provided by the repository and the script in README is used.

**Comment by Han-Jia** [comment]
how about the performance of the learned model on the tieredImageNet?

We obtain the dataset from the [link](https://drive.google.com/file/d/1nVGCTd9ttULRXFezh4xILQ9lUkg0WZCG/view) given by [MetaOptNet ](https://github.com/kjunelee/MetaOptNet)repo.
